config/locales/custom/general.yml

```diff
@@ -0,0 +1,10 @@
+en:
+  polls:
+    answers:
+      create:
+        success_notice: "Poll saved successfully!"
+      form:
+        submit_button: "Send"
```

Comment:
Not sure about this text either :thinking:, since in a person's mind "Send" is usually associated with emails instead of forms. Maybe something containing the word "vote" :thinking:.

Reply:
I used very simple texts waiting for customer opinion, probably they will want to change whatever we suggest now.